src/components/linear-progress/linear-progress.tsx

```diff
@@ -24,76 +38,56 @@ export class LinearProgress {
     @Prop({ reflect: true })
     public indeterminate: boolean = false;
 
+    /**
+     * A label used to describe the purpose of the element to users
+     * of assistive technologies, like screen readers.
+     * If not provided, the generic word of "Progress bar" will be used.
+     */
+    @Prop({ reflect: true })
+    public accessibleLabel?: string;
+
     @Element()
     private host: HTMLLimelLinearProgressElement;
 
-    private mdcLinearProgress: MDCLinearProgress;
-
-    public connectedCallback() {
-        this.initialize();
-    }
-
-    public componentDidLoad() {
-        this.initialize();
-    }
-
-    private initialize() {
-        const element = this.host.shadowRoot.querySelector(
-            '.mdc-linear-progress',
-        );
-        if (!element) {
-            return;
-        }
-
-        this.mdcLinearProgress = new MDCLinearProgress(element);
-        this.mdcLinearProgress.progress = this.value;
-    }
-
-    public disconnectedCallback() {
-        if (this.mdcLinearProgress) {
-            this.mdcLinearProgress.destroy();
-        }
-    }
-
     public render() {
         if (!this.isFinite(this.value)) {
             return;
         }
 
-        const classList = {
-            'mdc-linear-progress': true,
-            'mdc-linear-progress--indeterminate': this.indeterminate,
-        };
+        const loadingText = translate.get('loading', this.language);
+        const ariaLabel = translate.get('progress-bar', this.language);
+        const ariaValueNow = this.indeterminate ? undefined : this.value;
+        const ariaValueText = this.indeterminate ? loadingText : undefined;
 
         return (
-            <div
+            <Host
                 role="progressbar"
-                class={classList}
-                aria-label="Progress Bar"
+                aria-label={this.accessibleLabel || ariaLabel}
+                aria-live="polite"
                 aria-valuemin="0"
                 aria-valuemax="1"
-                aria-valuenow={this.value}
+                aria-valuenow={ariaValueNow}
+                aria-valuetext={ariaValueText}
+                style={{ '--percentage': `${this.value * PERCENT}%` }}
             >
-                <div class="mdc-linear-progress__buffer">
-                    <div class="mdc-linear-progress__buffer-bar"></div>
-                </div>
-                <div class="mdc-linear-progress__bar mdc-linear-progress__primary-bar">
-                    <span class="mdc-linear-progress__bar-inner" />
-                </div>
-                <div class="mdc-linear-progress__bar mdc-linear-progress__secondary-bar">
-                    <span class="mdc-linear-progress__bar-inner" />
-                </div>
-            </div>
+                <div class="progress" />
```

Comment:
Perhaps we should use the `meter` element here :thinking: 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meter

Reply:
Or `progress` :smile: 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress

Reply:
hmm… the `meter` element looks really interesting and feature-rich. But I don't actually know which one is better to use to be honest. 

Also, they seem to be hard to style, and styling support is inconsistent across browsers. 

If we decide to go that route, I'd say we can do a new refactor PR again later.

Reply:
Yeah I have no idea if it's possible to style them :smile: Just thought it could be good to have it for semantic reasons and to maybe assist screen readers etc

Reply:
I totally agree. But I have added so many accessibility things here just for that ;) So we're safer than before